shared/domains/chelonia/internals.js

```diff
@@ -1124,26 +1126,29 @@ export default (sbp('sbp/selectors/register', {
       if (latest !== recent) {
         console.debug(`[chelonia] Synchronizing Contract ${contractID}: our recent was ${recent || 'undefined'} but the latest is ${latest}`)
         // TODO: fetch events from localStorage instead of server if we have them
-        const events = await sbp('chelonia/out/eventsAfter', contractID, recent || contractID)
+        const eventsStream = sbp('chelonia/out/eventsAfter', contractID, recent || contractID)
```

Comment:
I think the code for `'chelonia/latestContractState'` also needs to be updated to use this this new method, right?